### PR TITLE
[APB-1300][LS,AK] Updated responseDate to be LocalDate as we are not

### PR DESCRIPTION
**Pull request by arkowalski**
concerned with the time. Added the responseDate to the relationship connector to be passed over

### CODE REVIEW

**Comment by arkadiusz-hmrc** [comment]
wait with the merge of this one, waiting for something to be merged before this APB-1344

File: app/uk/gov/hmrc/agentfiinvitation/controllers/package.scala [hunk 1]
```diff
@@ -18,8 +18,7 @@ package uk.gov.hmrc.agentfiinvitation
```
Comment (Smudgey) [inline app/uk/gov/hmrc/agentfiinvitation/controllers/package.scala]: This entire package doesn't seem to be used, so it can probably go

Comment (JTY009) [reply app/uk/gov/hmrc/agentfiinvitation/controllers/package.scala]: But there is a story that may require this to auth backend endpoints

Comment (JTY009) [reply app/uk/gov/hmrc/agentfiinvitation/controllers/package.scala]: fair enough

File: app/uk/gov/hmrc/agentfiinvitation/controllers/InvitationController.scala [hunk 2]
```diff
@@ -77,31 +71,32 @@ class InvitationController @Inject()(authConnector: AuthConnector,
...
-        Future.successful(BadRequest("Invitation ref can not be more then 100 characters in length"))
-      }
+      } else Future.successful(BadRequest("Invitation ref can not be more then 100 characters in length"))
     }
   }
 
+  protected def timeStamp = LocalDateTime.now.toString
+
   def acceptInvitation(clientId: String, invitationId: String): Action[AnyContent] = Action.async { implicit request =>
+    val responseDate = timeStamp
```
Comment (PaulHodgson) [inline app/uk/gov/hmrc/agentfiinvitation/controllers/InvitationController.scala]: maybe call this 'responseDateTime' - also in the repo & anywhere else?

File: app/uk/gov/hmrc/agentfiinvitation/repositories/InvitationMongoRepository.scala [hunk 3]
```diff
@@ -68,12 +66,11 @@ class InvitationMongoRepository @Inject()(mongoComponent: ReactiveMongoComponent
...
 
-  def updateInvitation(clientId: String, invitationId: String, status: String)(implicit ec: ExecutionContext): Future[Unit] = {
-    val responseDate = LocalDateTime.now()
+  def updateInvitation(clientId: String, invitationId: String, status: String, responseDate: String)(implicit ec: ExecutionContext): Future[Unit] = {
       atomicUpdate(BSONDocument("invitationId" -> invitationId),
         BSONDocument("$set" ->
-        BSONDocument("status" -> status)) ++
-        BSONDocument("$set" -> BSONDocument("responseDate" -> responseDate.toString)))
+          BSONDocument("status" -> status)) ++
+          BSONDocument("$set" -> BSONDocument("responseDate" -> responseDate)))
```
Comment (PaulHodgson) [inline app/uk/gov/hmrc/agentfiinvitation/repositories/InvitationMongoRepository.scala]: maybe call the val 'responseDateTime' but too big a change to justify renaming the mongo collection property

File: test/uk/gov/hmrc/agentfiinvitation/controllers/InvitationControllerSpec.scala [hunk 4]
```diff
@@ -38,7 +37,10 @@ class InvitationControllerSpec extends PlaySpec with MockitoSugar with GuiceOneA
   val mockRelationshipConnector: RelationshipConnector = mock[RelationshipConnector]
   val mockAuditService: AuditService = mock[AuditService]
   val mockAuthConnector: AuthConnector = mock[AuthConnector]
-  val controller = new InvitationController(mockAuthConnector, mockAuditService, mockMongoService, mockRelationshipConnector)
+  val controller = new InvitationController(mockAuthConnector, mockAuditService, mockMongoService, mockRelationshipConnector) {
+    override def timeStamp: String = "2017-09-21T14:00:03.986"
```
Comment (PaulHodgson) [inline test/uk/gov/hmrc/agentfiinvitation/controllers/InvitationControllerSpec.scala]: assign this def to the val testTimeStamp in the package object? Would make the assertion clearer and DRY

**Pull request merged**